src/pages/lib/graphqlapi/subscribe-data/q/platform/[platform].mdx

```diff
@@ -2,3 +2,455 @@ export const meta = {
+
+Before you begin, you will need:
+- An application connected to the API
+- Data already created to modify
+
+## Set up a real-time subscription
+
+Subscriptions is a GraphQL feature that allows the server to send data to its clients when a specific event happens. For example, you can subscribe to an event when a new record is created, updated, or deleted via the API. You can enable real-time data integration in your app with a subscription.
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Amplify, API } from 'aws-amplify';
+import { GraphQLSubscription } from '@aws-amplify/api';
+import * as subscriptions from './graphql/subscriptions';
+import { OnCreateTodoSubscription,OnUpdateTodoSubscription,OnDeleteTodoSubscription } from './API';
+
+// Subscribe to creation of Todo
+const createSub = API.graphql<GraphQLSubscription<OnCreateTodoSubscription>>(
+  { query: subscriptions.onCreateTodo }
+).subscribe({
+  next: ({ provider, value }) => console.log({ provider, value }),
+  error: (error) => console.warn(error)
+});
+
+// Subscribe to update of Todo
+const updateSub = API.graphql<GraphQLSubscription<OnUpdateTodoSubscription>>(
+  { query: subscriptions.onUpdateTodo }
+).subscribe({
+  next: ({ provider, value }) => console.log({ provider, value }),
+  error: (error) => console.warn(error)
+});
+
+// Subscribe to deletion of Todo
+const deleteSub = API.graphql<GraphQLSubscription<OnDeleteTodoSubscription>>(
+  { query: subscriptions.onDeleteTodo }
+).subscribe({
+  next: ({ provider, value }) => console.log({ provider, value }),
+  error: (error) => console.warn(error)
+});
+// Stop receiving data updates from the subscription
+createSub.unsubscribe();
+updateSub.unsubscribe();
+deleteSub.unsubscribe();
+```
+
+</Block>
+
+<Block name="JavaScript">
+
+```js
+import { Amplify, API } from 'aws-amplify';
+import * as subscriptions from './graphql/subscriptions';
+
+// Subscribe to creation of Todo
+const createSub = API.graphql(
+  { query: subscriptions.onCreateTodo }
+).subscribe({
+  next: ({ provider, value }) => console.log({ provider, value }),
+  error: (error) => console.warn(error)
+});
+
+// Subscribe to update of Todo
+const updateSub = API.graphql(
+   { query: subscriptions.onUpdateTodo }
+  ).subscribe({
+    next: ({ provider, value }) => console.log({ provider, value }),
+    error: (error) => console.warn(error)
+  });
+
+// Subscribe to deletion of Todo
+const deleteSub = API.graphql(
+  { query: subscriptions.onDeleteTodo }
+).subscribe({
+  next: ({ provider, value }) => console.log({ provider, value }),
+  error: (error) => console.warn(error)
+});
+
+// Stop receiving data updates from the subscription
+createSub.unsubscribe();
+updateSub.unsubscribe();
+deleteSub.unsubscribe();
+```
+
+</Block>
+</BlockSwitcher>
+
+When using **AWS AppSync** subscriptions, be sure that your AWS AppSync configuration is at the root of the configuration object, instead of being under the API:
```

Comment:
@renebrandel Line 101 [TODO]: Note from last review "I'm actually not sure if this is true anymore." Needs follow up on if this should be removed or updated.